docs/src/main/sphinx/connector/query-table-function-ordering.fragment

```diff
@@ -0,0 +1,4 @@
+.. note::
+
+    If the table function contains a query with an ``ORDER BY`` clause, the
+    function result may not be ordered as expected.
```

Comment:
I would move this fragment to the bottom, so that for each connector it comes after the example SQL snippet.

Maybe we could reword it along @martint's suggestion, e.g.: "The query engine does not preserve the order of the results of this function. If the passed query contains an ``ORDER BY`` clause, the function result may not be ordered as expected."
Otherwise, this comment looks like admitting that we have a bug, while this is just how the engine is supposed to work.

Reply:
Done